apps/dialtone-documentation/docs/.vuepress/baseComponents/AllTokens.vue

```diff
@@ -14,50 +52,243 @@
       :options="THEMES"
       @change="setTheme"
     />
-  </dt-stack>
-  <div v-for="category in categories" :key="category">
-    <slot :name="category">
-      <!-- slot for anchor header -->
-    </slot>
-    <token-table :category="category" :format="format" :theme="theme" />
+  </div>
+  <div v-for="category in Object.keys(filteredTokens)" :key="category">
+    <h2
+      :id="category"
+      class="d-docsite--header-2 d-tt-capitalize"
+      tabindex="-1"
+    >
+      <a
+        :href="`#${category}`"
+        class="header-anchor d-link d-docsite--link d-link"
+      >#</a>
+      {{ category }}
+    </h2>
+    <p class="dialtone-intro">
+      {{ categoryIntroAndDesc[category]?.intro }}
+    </p>
+    <div v-html="categoryIntroAndDesc[category]?.description" />
+    <token-table :category="category" :tokens="filteredTokens[category]" />
+  </div>
+  <div
+    v-if="!hasSearchResults"
+    class="d-d-flex d-fl-center d-p16 d-gg4 d-fc-tertiary d-fs-300"
+  >
+    <span>No results found for</span>
+    <strong class="d-fw-semibold">
+      &OpenCurlyDoubleQuote;{{ searchCriteria }}&CloseCurlyDoubleQuote;
+    </strong>
   </div>
 </template>
 
-<script>
+<script setup>
+import * as tokensJson from '@dialpad/dialtone-tokens/dist/doc.json';
```

Comment:
`import tokensJson from '@dialpad/dialtone-tokens/dist/doc.json';`
This should be the same.